site/app/templates/calendar/NewCalendarItem.twig

```diff
@@ -5,16 +5,31 @@
     <p class="std-margin">
         Please select the type of item, when it should be, and the text it should display.
     </p>
-    <label for="calendar-item-type">Message Type</label>
-    <select id="calendar-item-type" name="type">
-        <option value="note">Note</option>
-        <option value="announcement">Announcement</option>
-        <option value="important">Important</option>
-    </select><br>
-    <label for="calendar-item-text">Message Text</label>
-    <input type="text" id="calendar-item-text" name="text"><br>
-    <label for="edit-picker">Date</label>
-    <input id="edit-picker" class="edit-picker" type="text" name="date">
+
+    <style>
+        div.calendar-submit {
+            display: grid;
+            grid-template-columns: max-content max-content;
+            grid-gap: 5px; 
+        }
+        div.calendar-submit label { text-align: "right"; }
+        div.calendar-subm1t label:after { content: ":"; }
+    </style>
```

Comment:
CSS like this should be moved to the relevant CSS file.  `site/public/css/calendar.css` would probably be an appropriate place for this.

Reply:
Gotcha, have moved all relevant portions to the CSS file!